src/index.css

```diff
@@ -1,10 +1,40 @@
+:root {
+  --bg-color: #ffffff;
+  --text-color: #333333;
+  --input-bg: #ffffff;
+  --input-border: #dddddd;
+  --button-bg: #4CAF50;
+  --button-hover: #45a049;
+  --delete-button-bg: #f44336;
+  --delete-button-hover: #d32f2f;
+  --todo-bg: #ffffff;
+  --todo-shadow: rgba(0, 0, 0, 0.1);
+}
+
+@media (prefers-color-scheme: dark) {
```

Comment:
This is almost there, but we need to add another selector for when the document element has data-theme="dark" etc.

Reply:
I realised we made the changes in index.css but we're not importing this anywhere! We should add this to src/main.tsx

Reply:
Please remove the code from index.css that is centering the to do list in the middle of the page.